.github/workflows/chromatic.yml

```
@@ -0,0 +1,23 @@
+name: "Chromatic"
+
+on:
+    pull_request:
+        types: [ labeled ]
```

If we add the synchronize label here, we can run the chromatic action whenever a new commit is added, and the label is present. That might be a better default?
```suggestion
        types: [labeled, synchronize]
```

---

This (and the condition check on the job below) is exactly what I was looking for. Thanks.